bin/hbase

```diff
@@ -188,7 +188,7 @@ if [ -d "$HBASE_HOME/hbase-server/target/hbase-webapps" ]; then
 fi
 
 #add the hbase jars for each module
-for f in $HBASE_HOME/hbase-jars/hbase*.jar; do
+for f in $HBASE_HOME/lib/hbase*.jar; do
```

Comment:
Looks like the commit only fixes the tar-ball version of shell and not a dev build (where the lib/ is not populated). Curious if that was intentional. Fix lgtm otherwise.